app/templates/components/new-course.hbs

```diff
@@ -1,16 +1,29 @@
 <section class='newcourse'>
-  <label>
-    {{t 'general.title'}}
-    {{input value=course.title placeholder=placeholder}}
-  </label>
-  <label>
-    {{t 'general.academicYear'}}
-    {{#action-menu title=currentAcademicYear icon=false}}
-      {{#each year in years}}
-        <li {{action 'setYear' year}}>{{year.academicYearTitle}}</li>
-      {{/each}}
-    {{/action-menu}}
-  </label>
-  <button class='cancel' {{action 'cancel'}}>{{t 'general.cancel'}}</button>
-  <button class='save' {{action 'save'}}>{{t 'general.save'}}</button>
+  <div class="newcourse-title">New Course</div>
```

Comment:
This will need to be in a translation.  We can commit this code, just make sure there is a new ticket created to translate 'New Course'